model/gen-ai/spans.yaml

```diff
@@ -82,9 +85,6 @@ groups:
     attributes:
       - ref: gen_ai.request.model
         requirement_level: required
-      - ref: gen_ai.openai.request.seed
```

Comment:
I didn't check if it was released, but if it was we need to add this to the deprecated files as well.

Reply:
@joaopgrassi this was done at https://github.com/open-telemetry/semantic-conventions/pull/1710, and this PR is a follow up for https://github.com/open-telemetry/semantic-conventions/pull/1710 do more clean up.